app/tests/components/Form/ApplicationForm.test.tsx

```diff
@@ -26,6 +26,29 @@ const mockQueryPayload = {
       id: 'TestApplicationID',
       formData: {},
       status: 'draft',
+      updatedAt: '2022-09-12T14:04:10.790848-07:00',
+    };
+  },
+  Query() {
+    return {
+      openIntake: {
+        closeTimestamp: '2022-08-27T12:51:26.69172-04:00',
+      },
+    };
+  },
+};
+
+const submissionPayload = {
```

Comment:
```suggestion
const submissionPayload = {
  ...mockQueryPayload,
```

This way you don't need to copy the `Query()` function every time

Reply:
Man you're being way too opinionated. This is getting annoying.

Reply:
In what way proposing solutions following best practices to reduce code duplication is being opinionated?

Reply:
I mean most of this review in general.

Reply:
Feel free to discuss any feedback on the relevant threads. I'm happy to expand on any of my suggestions. Some of them are probably beyond the scope of this PR and should be tracked in a separate issue